android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -724,7 +733,7 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
             Log.d(TAG, "submit url after a short delay for starting snowflake")
             ioScope.launch() {
                 Log.d(TAG, "start snowflake delay")
-                delay(10000L) // wait 10 seconds
+                delay(1000L) // wait 1 second
```

Comment:
that's a big change 👍

Reply:
i believe @stevenmcdonald said snowflake took less time than the other services to get ready?